AK/Types.h

```diff
@@ -55,20 +55,6 @@ using int32_t = i32;
 using int64_t = i64;
 
 using pid_t = int;
-
-#else
-#    include <stddef.h>
-#    include <stdint.h>
-#    include <sys/types.h>
-
-#    ifdef __ptrdiff_t
-using __ptrdiff_t = __PTRDIFF_TYPE__;
-#    endif
-
-#    if defined(AK_OS_WINDOWS)
-using ssize_t = AK::Detail::MakeSigned<size_t>;
-using mode_t = unsigned short;
```

Comment:
This for sure breaks Jakt on windows, does it not? We need that for some headers to not choke

Reply:
In case types we defined in AK/Types.h happen to match what system headers define, there won't be any problem as far as compilers are concerned. And in fact, the PR does not break Jakt (https://github.com/SerenityOS/jakt/pull/1500) and CI here is green.